front/src/styles/common.css

```diff
@@ -4,35 +4,61 @@
   font-weight: 600;
   font-size: 14px;
   font-family: Open Sans;
+  font-style: normal;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 h1 {
   font-size: 54px;
+  line-height: 92px;
```

Comment:
what is it for?

Reply:
I took it from the layout, it's all set up like this. what exactly is needed is the height of the text line, without it the text becomes slightly lower